tests/services/phase.service.spec.ts

```diff
@@ -13,7 +13,7 @@ const moderationPhaseSettingsFile: {} = {
 
 const invalidPhasesSettingsFile: {} = {
   ...moderationPhaseSettingsFile,
-  'openPhases': ['dummyPhase']
+  'openPhases': []
```

Comment:
Can I clarify why this has to be updated? Is it because the test says `no openPhases are defined`?

Reply:
Yes, initially no openPhases could mean a "dummyphase" but in our new checks that would throw an error in the undefined open phase area instead of no open phases.